compiler/src/iree/compiler/Codegen/Common/TransformExtensions/CommonExtensionsOps.td

```diff
@@ -153,4 +153,50 @@ def ForeachThreadToWorkgroupOp : Op<Transform_Dialect,
   }];
 }
 
+def TileToWorkgroupsOp :
+    Op<Transform_Dialect, "iree.tile_to_workgroups_op",
+      [AttrSizedOperandSegments,
+       DeclareOpInterfaceMethods<MemoryEffectsOpInterface>,
+       TransformOpInterface]> {
+  let description = [{
+    Wrapper around `structured.tile_to_foreach_thread_op` for use within IREE.
```

Comment:
"Please see the doc of tile_to_foreach_thread_op in upstream MLIR for a full description of the op semantics".